moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -801,9 +816,9 @@ bool JogCalcs::updateJoints()
         }
         else
         {
-          ROS_ERROR_STREAM_ONCE_NAMED(LOGNAME, "An acceleration limit is not defined for this joint, '"
-                                                   << joint_model->getName() << "'; minimum stop distance should not "
-                                                                                "be used for collision checking");
+          ROS_WARN_STREAM_THROTTLE_NAMED(ROS_LOG_THROTTLE_PERIOD, LOGNAME,
```

Comment:
This might be kind of a dumb question, but where is ROS_LOG_THROTTLE_PERIOD defined?

We definitely don't want to print this very often, it gets really annoying (we found that out first-hand). I would say, like, once every ten seconds or so. Or just once

Reply:
Can you please move the definition of ROS_LOG_THROTTLE_PERIOD up by LOGNAME (line 46). We're trying to put all the constants at the top of the .cpp files now so they are easy to find

Reply:
It was defined in the header for jog arm parameters. I moved it to the jog calcs cpp and the collision check cpp